package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "cordova-plugin-splashscreen",
-  "version": "5.0.4-dev",
+  "name": "dc-cordova-plugin-splashscreen",
```

Comment:
This change (and the version) has to be removed if this should be merged to the original Cordova plugin.

Reply:
I will remove it.